### ISSUE TITLE
Dynamic Windows Name Trouble

### ISSUE THREAD

**Issue by natemcandrews** [pywinauto/pywinauto#1060]
## Expected Behavior
After opening Spotify and playing a song, I should still be able to click buttons and navigate Spotify

## Actual Behavior
If a song starts playing, the name of the Spotify pane changes to the song that is playing and I receive
pywinauto.findbestmatch.MatchError: Could not find 'SpotifyPremium' in 'dict_keys(['Pane', 'Bryce Vine - Sour Patch Kids', 'Bryce Vine - Sour 
Patch KidsPane'])'

note: Starting a song or opening a playlist is possible if a song is not playing

## Steps to Reproduce the Problem

  1. Click any buttons in Spotify using pywinauto
  2. Play a song
  3. While the song is playing, the name of Spotify changes to the current song

## Short Example of Code to Demonstrate the Problem

        app = application.Application(backend="uia")
        app.start(r"C:/Users/Natem/AppData/Roaming/Spotify/spotify.exe", timeout = 5)
        t.sleep(6)
        playlist = app.SpotifyPremium.child_window(title='bops', found_index = 0).invoke()
        play = app.SpotifyPremium.child_window(title = 'Play',  found_index = 0).invoke()
        t.sleep(2)
        playlist = app.SpotifyPremium.child_window(title='night vibes', found_index = 0).invoke()

## Specifications

  - Pywinauto version: 0.6.8
  - Python version and bitness: Python38-32 64bit
  - Platform and OS: Windows 10 64 bit

**Comment by vasily-v-ryabov** [comment]
If it is for main window only, it's easy to bypass:
```python
main_window = app.window(control_type="Pane")
main_window.child_window(title='night vibes', found_index = 0).invoke()
```
The first level of search takes only top level windows into account. Usually there is only 1 top level window that can be checked by `app.windows()`.

**Comment by vasily-v-ryabov** [comment]
Also you can use `app.wait_cpu_usage_lower()` instead of hard-coded time sleeps or more precise waiting for window specification. See [Waiting for Long Operations](https://pywinauto.readthedocs.io/en/latest/wait_long_operations.html).

**Comment by natemcandrews** [comment]
Yes! that worked perfectly. I appreciate the help. I gotta say too, I have noticed that you are super good at responding to questions and helping us who are still learning pywinauto. Keep up the great work!

**Comment by vasily-v-ryabov** [comment]
Thanks! Not super good. Several issues are still in unread (some are few months old). You're lucky today. :)

**Comment by alx-xlx** [comment]
how can we get the Title of the window that dynamically changes ??

**Comment by ctechn** [comment]
> how can we get the Title of the window that dynamically changes ?? in the above solution there still is title being passed

I have the same issue, any resolution? We need pywinauto to work without depending on window title (or class, since it's a Chrome web widget)
